native/src/main/java/io/ballerina/stdlib/sql/nativeimpl/ExecuteProcessor.java

```diff
@@ -184,21 +190,20 @@ public static Object nativeBatchExecute(Environment env, BObject client, BArray
             Future balFuture = env.markAsync();
             SQL_EXECUTOR_SERVICE.execute(()-> {
                 Object resultStream =
-                        nativeBatchExecuteExecutable(client, paramSQLStrings, statementParameterProcessor, generateKeys,
+                        nativeBatchExecuteExecutable(client, paramSQLStrings, statementParameterProcessor,
```

Comment:
This native batch execute method can be now removed, this is now redundant, isn't it?

Reply:
Fixed in 8bcd42e